rules/X++/IDisposableWithoutDispose.xq

```diff
@@ -0,0 +1,24 @@
+(: Find classes that implement IDisposable, but do not have a Dispose method :)

```

Comment:
I would expect the compiler to also error on this. It is basic check for interface implementation by a class.